### PR TITLE
Adds an @include_columns option to the GSheetToCsv class.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Need to add tests still, though the lack of existing tests for this class makes that considerably harder.

File: pegasus/rake/seed.rake [hunk 1]
```diff
@@ -166,14 +167,20 @@ class GSheetToCsv
       puts "Error on file: #{@gsheet_path}, #{e}"
       throw e
     end
-    if @exclude_columns.empty?
+    if @include_columns.empty? && @exclude_columns.empty?
```
Comment (ashercodeorg) [inline pegasus/rake/seed.rake]: Is there benefit to having the if-else? The execution will be more efficient (not that we care?), at the cost of adding code complexity.

Comment (philbogle) [inline pegasus/rake/seed.rake]: This seems fragile, better not to have it. (e.g. we could add new factors that influence the output other than @include_columns and @exclude_columns)

**Comment by philbogle** [comment]
LGTM.  Up to you rather you want to get rid of the if-else now.

**Comment by ashercodeorg** [comment]
Removed the "if". Need to write tests still, which I expect I'll need help with.

**Comment by philbogle** [comment]
As a security improvement to previously untested code, it is OK to check this in if it passes adhoc tests and then do the tests as a followup.

For the testing, I think we should factor out the guts of the rake test and write unit tasks for that, which shouldn't be too painful.  The additional benefit of integration testing the rake task probably doesn't justify the opportunity cost of the time required to add it.

**Pull request merged**